headhunter/data/mail_template.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="crm_signature_agreem" model="mail.template">
+        <field name="name">CRM : Signature Agreement</field>
+        <field name="model_id" ref="crm.model_crm_lead"/>
+        <field name="email_from">{{ user.email_formatted }}</field>
+        <field name="email_to">{{ object.partner_id.email }}</field>
+        <field name="auto_delete" eval="False"/>
+        <field name="subject">Dear customer, please sign this service agreement.</field>
+        <field name="body_html">
+            <![CDATA[
+                <p style="margin:0px 0 16px 0;box-sizing:border-box;">Hello, here is a link to sign our
+                    service agreement.<br/><br/>It will provide details on our offer and what
+                    to expect from our collaboration.<br/><br/>
+                </p>
+                <div style="text-align: center; padding: 16px 0px 16px 0px; font-size: 14px;">
+                    <a class="btn btn-fill-primary btn-lg"
+                    t-att-href="'/sign/document/mail/' + str(env.ref('headhunter.sign_request_1').id) + '/'+ str(env.ref('headhunter.sign_request_1').request_item_ids.access_token) + '?timestamp=' + str(env.ref('headhunter.sign_request_1').request_item_ids._generate_expiry_link_timestamp()) + '&exp=' + str(env.ref('headhunter.sign_request_1').request_item_ids._generate_expiry_signature(env.ref('headhunter.sign_request_1').request_item_ids.id, int(env.ref('headhunter.sign_request_1').request_item_ids._generate_expiry_link_timestamp())))"
+                        style="border-style:solid;box-sizing:border-box;transition-property:none;transition-delay:0s;transition-timing-function:ease;transition-duration:0s;transition-behavior:normal;border-left-color:#714b67;border-bottom-color:#714b67;border-right-color:#714b67;border-top-color:#714b67;border-left-width:1px;border-bottom-width:1px;border-right-width:1px;border-top-width:1px;background-color:#714b67;user-select:none;cursor:pointer;vertical-align:middle;text-align:center;line-height:1.5;font-weight:500;display:inline-block;color:#ffffff;padding: 8px 16px; text-decoration: none; border-radius: 5px; font-size: 13px;">
+                        Service Agreement</a>
+                </div>
+                <p style="box-sizing:border-box;margin-bottom: 0px;"> Have a nice day,</p>
+                <p style="margin:0px 0 16px 0;box-sizing:border-box;">
+                    <br/>
+                </p>
+            ]]>
+        </field>
+    </record>
+    <record id="mail_template_48" model="mail.template">
```

Comment:
Couldn't you update email_template_data_applicant_interest instead? Both look very similar